src/components/MessagesList/MessagesGroup/MessagesGroup.vue

```diff
@@ -140,6 +140,9 @@ export default {
 			} else if (this.messages[0].lastEditActorId === this.$store.getters.getActorId()
 				&& this.messages[0].lastEditActorType === this.$store.getters.getActorType()) {
 				return t('spreed', '(edited by you)')
+			} else if (this.lastEditActorId === 'deleted_users'
```

Comment:
Either I'm missing something, and it's not implemented, or something else, but I'm still receiving deleted user credentials from server: 
![image](https://github.com/nextcloud/spreed/assets/93392545/bdbe34fd-fed7-4734-816b-d95f238fc897)

Reply:
And for system messages now as well:
![image](https://github.com/nextcloud/spreed/assets/93392545/e54aac90-fb81-4edc-9c87-4ad67889080b)

Reply:
Did you delete the user? Or just remove them from the room?

For system messages nothing changed and that was handled correctly already.

Reply:
`occ user:delete user6`

Reply:
Can you try deleting via web? Maybe CLI has been broken by someone not loading all apps anymore ...

Reply:
Tried both occ and web deletion, doesn't make any difference
But noticed, that after server restart / db upgrade previously deleted users dissapear. Maybe the value of `getActorInformation` for specific user is cached somewhere, when PHP runs it?

![image](https://github.com/nextcloud/spreed/assets/93392545/9497e6b7-2322-4064-9497-f040bb050128)

Reply:
the display name is cached, but that should expire on deletion or after 5mins

Reply:
So let's merge it?